wfe/wfe.go

```diff
@@ -1840,8 +1840,22 @@ func (wfe *WebFrontEndImpl) orderForDisplay(
 	defer order.RUnlock()
 
 	// Copy the initial OrderRequest from the internal order object to mutate and
-	// use as the result.
-	result := order.Order
+	// use as the result. We have to make sure to copy the identifiers and
+	// authorizations *slices* and not *pointers to the slices* or the mutations
```

Comment:
A more correct way to phrase this would be that we have to copy the _contents_ of the idents and authzs slices, rather than copying the slices themselves, because slices are just pointers with some extra metadata.

Those fields aren't "pointers to slices" (that would be `type *[]string`).